packages/manager/tools/testcafe/utils/helpers.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-undef */
```

Comment:
Not necessary I think.

```suggestion
```

Reply:
it's necessary, because we use variables from navigator, such "window" ...

Reply:
Strange this isn't already handled by our eslint config